### ISSUE TITLE
PermutationImportance fit doesn't use sample_weight

### ISSUE THREAD

**Issue by CouthuresJeremy** [TeamHG-Memex/eli5#314]
I am using Keras Model, LightGBM LGBMClassifier, XGBoost XGBClassifier on default tree_method and hist tree_method. Also, my weights include negative weights.

I tried to see the impact of my weights on PermutationImportance weights. 
It is giving me exactly the same results with and without weights.

This is mainly how I dot it:

import lightgbm as lgb
gbm = lgb.LGBMClassifier(class_weight="balanced")
gbm.fit(X_train, y_train)

import eli5
from eli5.sklearn import PermutationImportance
perm_gbm = PermutationImportance(gbm, scoring="roc_auc").fit(X_test, y_test, sample_weight=weights_test)
display(eli5.show_weights(perm_gbm))

**Comment by aghoshpub** [comment]
Should be an easy extension [here](https://github.com/TeamHG-Memex/eli5/blob/master/eli5/permutation_importance.py#L86) to include weights for a score function. Most score functions already support weights.
